ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/OsakaTargetingGasLimitCalculator.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright contributors to Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.mainnet;
+
+import org.hyperledger.besu.datatypes.BlobGas;
+import org.hyperledger.besu.datatypes.Wei;
+import org.hyperledger.besu.ethereum.mainnet.feemarket.BaseFeeMarket;
+import org.hyperledger.besu.evm.gascalculator.GasCalculator;
+
+public class OsakaTargetingGasLimitCalculator extends CancunTargetingGasLimitCalculator {
+
+  /** The blob base cost constant for Osaka */
+  private static final long BLOB_BASE_COST = 1 << 14; // 2^14
```

Comment:
This belongs to the `OsakaGasCalculator`. My understand is that It needs to override `BLOB_GAS_PER_BLOB` in `CancunGasCalculator`

Reply:
CancunGasCalculator.BLOB_GAS_PER_BLOB is GAS_PER_BLOB from 4844? https://eips.ethereum.org/EIPS/eip-4844#parameters

I assumed they were different since in this line from 7918 it uses both -
`    if BLOB_BASE_COST * parent.base_fee_per_gas > GAS_PER_BLOB * get_base_fee_per_blob_gas(parent):`

Reply:
also GAS_PER_BLOB is bigger: 2**17

Reply:
yeah, you are correct. My bad!